CRITICAL - 2024-05-27 02:35:27 --> Error: Class "app\Models\miModelo" not found
[Method: GET, Route: /]
in APPPATH\Controllers\Home.php on line 12.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Home->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-05-27 02:37:21 --> Error: Class "app\Models\miModelo" not found
[Method: GET, Route: /]
in APPPATH\Controllers\Home.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Home->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-05-27 02:37:23 --> Error: Class "app\Models\miModelo" not found
[Method: GET, Route: /]
in APPPATH\Controllers\Home.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Home->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-05-27 02:42:33 --> Error: Class "app\Models\miModelo" not found
[Method: GET, Route: /]
in APPPATH\Controllers\Home.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Home->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-05-27 02:47:30 --> Error: Class "app\Models\miModelo" not found
[Method: GET, Route: /]
in APPPATH\Controllers\Home.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Home->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-05-27 03:37:53 --> ParseError: syntax error, unexpected variable "$contador"
[Method: GET, Route: usuarios]
in APPPATH\Views\usuarios\user.php on line 18.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('usuarios/user', [], true)
 3 APPPATH\Controllers\Home.php(24): view('usuarios/user', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->usuarios()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-05-27 03:40:48 --> ErrorException: Undefined array key "dap_usuario"
[Method: GET, Route: usuarios]
in APPPATH\Views\usuarios\user.php on line 20.
 1 APPPATH\Views\usuarios\user.php(20): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "dap_usuario"', 'C:\\xampp\\htdocs\\SistemaNT\\app\\Views\\usuarios\\user.php', 20)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\SistemaNT\\app\\Views\\usuarios\\user.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('usuarios/user', [], true)
 5 APPPATH\Controllers\Home.php(24): view('usuarios/user', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->usuarios()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-05-27 16:55:41 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\SistemaNT\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\SistemaNT\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'bdd_sistemant', 3306, '', 0)
#1 C:\xampp\htdocs\SistemaNT\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\SistemaNT\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\SistemaNT\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT `u`.`usu...', Array, false)
#4 C:\xampp\htdocs\SistemaNT\app\Models\miModelo.php(22): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\SistemaNT\app\Controllers\Home.php(20): App\Models\miModelo->selectUsuarios()
#6 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(933): App\Controllers\Home->usuarios()
#7 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\SistemaNT\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\SistemaNT\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\SistemaNT\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\SistemaNT\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\SistemaNT\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\SistemaNT\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\SistemaNT\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT `u`.`usu...', Array, false)
#3 C:\xampp\htdocs\SistemaNT\app\Models\miModelo.php(22): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\SistemaNT\app\Controllers\Home.php(20): App\Models\miModelo->selectUsuarios()
#5 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(933): App\Controllers\Home->usuarios()
#6 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\SistemaNT\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\SistemaNT\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\SistemaNT\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-05-27 16:55:41 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: GET, Route: usuarios]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT `u`.`usu_usuario`, `u`.`usu_estado`, `d`.`dap_nombres`, `d`.`dap_correo`, `d`.`dap_cedula`
FROM `tbl_usuario` `u`
JOIN `tbl_datosp` `d` ON `d`.`dap_id`=`u`.`dap_id`
WHERE `u`.`usu_estado` = :u.usu_estado:', [...], false)
 3 APPPATH\Models\miModelo.php(22): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Home.php(20): App\Models\miModelo->selectUsuarios()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->usuarios()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-05-27 16:56:01 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\SistemaNT\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\SistemaNT\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'bdd_sistemant', 3306, '', 0)
#1 C:\xampp\htdocs\SistemaNT\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\SistemaNT\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\SistemaNT\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT `u`.`usu...', Array, false)
#4 C:\xampp\htdocs\SistemaNT\app\Models\miModelo.php(22): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\SistemaNT\app\Controllers\Home.php(20): App\Models\miModelo->selectUsuarios()
#6 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(933): App\Controllers\Home->usuarios()
#7 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\SistemaNT\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\SistemaNT\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\SistemaNT\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\SistemaNT\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\SistemaNT\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\SistemaNT\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\SistemaNT\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT `u`.`usu...', Array, false)
#3 C:\xampp\htdocs\SistemaNT\app\Models\miModelo.php(22): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\SistemaNT\app\Controllers\Home.php(20): App\Models\miModelo->selectUsuarios()
#5 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(933): App\Controllers\Home->usuarios()
#6 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\SistemaNT\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\SistemaNT\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\SistemaNT\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-05-27 16:56:01 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: GET, Route: usuarios]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT `u`.`usu_usuario`, `u`.`usu_estado`, `d`.`dap_nombres`, `d`.`dap_correo`, `d`.`dap_cedula`
FROM `tbl_usuario` `u`
JOIN `tbl_datosp` `d` ON `d`.`dap_id`=`u`.`dap_id`
WHERE `u`.`usu_estado` = :u.usu_estado:', [...], false)
 3 APPPATH\Models\miModelo.php(22): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Home.php(20): App\Models\miModelo->selectUsuarios()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->usuarios()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-05-27 16:56:08 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\SistemaNT\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\SistemaNT\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'bdd_sistemant', 3306, '', 0)
#1 C:\xampp\htdocs\SistemaNT\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\SistemaNT\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\SistemaNT\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT `u`.`usu...', Array, false)
#4 C:\xampp\htdocs\SistemaNT\app\Models\miModelo.php(22): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\SistemaNT\app\Controllers\Home.php(20): App\Models\miModelo->selectUsuarios()
#6 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(933): App\Controllers\Home->usuarios()
#7 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\SistemaNT\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\SistemaNT\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\SistemaNT\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\SistemaNT\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\SistemaNT\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\SistemaNT\system\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\SistemaNT\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT `u`.`usu...', Array, false)
#3 C:\xampp\htdocs\SistemaNT\app\Models\miModelo.php(22): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\SistemaNT\app\Controllers\Home.php(20): App\Models\miModelo->selectUsuarios()
#5 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(933): App\Controllers\Home->usuarios()
#6 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\SistemaNT\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\SistemaNT\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\SistemaNT\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\SistemaNT\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-05-27 16:56:08 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: GET, Route: usuarios]
in SYSTEMPATH\Database\BaseConnection.php on line 457.
 1 SYSTEMPATH\Database\BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT `u`.`usu_usuario`, `u`.`usu_estado`, `d`.`dap_nombres`, `d`.`dap_correo`, `d`.`dap_cedula`
FROM `tbl_usuario` `u`
JOIN `tbl_datosp` `d` ON `d`.`dap_id`=`u`.`dap_id`
WHERE `u`.`usu_estado` = :u.usu_estado:', [...], false)
 3 APPPATH\Models\miModelo.php(22): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Home.php(20): App\Models\miModelo->selectUsuarios()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->usuarios()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
